.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "docker"
```

Comment:
Why do we need this?

Reply:
Sorry, I deleted the wrong entry when copying from another repo. Corrected!